src/parser/bison_parser.y

```diff
@@ -914,6 +915,7 @@ operand:
 	|	extract_expr
 	|	cast_expr
 	|	array_expr
+	|	interval_expression
```

Comment:
I would suggest to introduce an 'inverval_literal' in ll. 1029 ('literal: [...]') instead. (interval_expression is used in the SQL standard to enforce type compatibility (see http://www.contrib.andrew.cmu.edu/~shadow/sql/sql1992.txt  p.168 ), which we do not do in the hyrise parser)

Reply:
I#m just about to change that. I guess Expr::isLiteral()`` should return true for interval literals, as well? And is there a reason not to include date literals here?

Reply:
For now, I added `isType(kExprLiteralDate) || isType(kExprLiteralInterval)` to `Expr::isLiteral()`.